pkg/kubelet/kuberuntime/kuberuntime_container_linux.go

```diff
@@ -177,10 +184,13 @@ func (m *kubeGenericRuntimeManager) configureContainerSwapResources(lcr *runtime
 	// NOTE(ehashman): Behavior is defined in the opencontainers runtime spec:
 	// https://github.com/opencontainers/runtime-spec/blob/1c3f411f041711bbeecf35ff7e93461ea6789220/config-linux.md#memory
 	switch m.memorySwapBehavior {
+	case kubelettypes.NoSwap:
+		swapConfigurationHelper.ConfigureNoSwap(lcr)
 	case kubelettypes.LimitedSwap:
 		swapConfigurationHelper.ConfigureLimitedSwap(lcr, pod, container)
 	default:
-		swapConfigurationHelper.ConfigureUnlimitedSwap(lcr)
+		klog.InfoS("NoSwap is set due to memorySwapBehavior not specified", "memorySwapBehavior", m.memorySwapBehavior)
```

Comment:
log this at startup, not for every container

Reply:
moved this to startup.